components/config/examples/generate_sublime.rs

```diff
@@ -26,8 +29,24 @@ fn main() {
             builder.add_from_folder(package_dir, true).unwrap();
             let ss = builder.build();
             dump_to_file(&ss, packpath_newlines).unwrap();
-
-            for s in ss.syntaxes() {
+            let syntaxes_sorted = ss
```

Comment:
you don't need itertools, you can just create a hashmap and append the extensions to a set and finally iter on the sorted hashmap

Reply:
I also tried to use HashMap, and it was not too difficult to iter on it nor sort it. 
The main difficult part for me about Rust syntax is really understanding of languages rules / syntax and I'm stuck at understanding how to create/update a file_extensions vector from existing ones.
And to be honest, I need a rest after hours of non succeeding in Rust so many times :) (without getting too much value from it)

Reply:
I came with a solution that do not need itertools and instead I'm making use of HashSet and BTreeMap (which ensures order, while HashMap was not helping here). I'll check how to fix the order, using something like sort_by, as for now the default order is not what I want when iterating on resulted collections.

Reply:
@Keats I've applied the suggestions. As you will notice, I got rid of useless dependencies, however code might probably be improved.
That's the best I could produce for now with my basic knowledge of Rust, which I've only started learning about for this pull request :)